scripts/tsup.ts

```diff
@@ -52,7 +53,8 @@ export function getConfig({
             `export * from '${srcTypesFile}'`,
           )
         }
-        validateExports(exports)
+        await validateExports(exports)
+        await generateProxyPackageEntrypoints(exports)
```

Comment:
Will check, but don't think we will need to do this during development (so can remove this line).